hosted_control_planes/index.adoc

```diff
@@ -12,14 +12,9 @@ include::modules/hosted-control-planes-overview.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hypershift-addon-intro[HyperShift add-on (Technology Preview)]
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/install_upgrade/adv_config_install#hypershift-addon-intro[HyperShift add-on (Technology Preview)]
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)]
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/hosted_intro#hosted-control-planes-intro[Hosted control planes for Red Hat OpenShift Container Platform (Technology Preview)]
```

Comment:
Similar comment from the control-plane.adoc file; I didnt see a section with the Red Hat OpenShift Container Platform title.

Reply:
Fixed!